pkgs/development/python-modules/pyshark/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchFromGitHub, py, lxml, pytestCheckHook, wireshark-cli }:
+
+buildPythonPackage rec {
+  pname = "pyshark";
+  version = "0.4.2.11";
+
+  src = fetchFromGitHub {
+    owner = "KimiNewt";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "07dkhkf85cplcj1h3k8mmqzsn4zdkxzr0zg3gvf8yc8p5g5azx9q";
+  };
+
+  propagatedBuildInputs = [
+    py
+    lxml
+  ];
+
+  preConfigure = "cd src";
+  preCheck = "cd ..";
```

Comment:
```suggestion
  preConfigure = ''
    cd src
  '';

  preCheck = ''
    cd ..
  '';
```

Reply:
Why should I change this?

Reply:
Because we had multiple occasions where in some script new lines where not appended correctly and this just illuminates this.

Reply:
`preConfigure` and `preCheck` are not appended to some script, but are set as environment variables and then evaluated by the `runHook` function of stdenv.

Reply:
Still better save than sorry and it is easier to just put all commands into ''...'' instead of differentiating.

Reply:
I don't see the point since it works fine and will always work fine, but if you insist I will change it.